cmd/imagestream.go

```diff
@@ -20,6 +23,9 @@ type ImageStreamCleanupOptions struct {
 	Namespace   string
 	Tag         bool
 	Sorted      string
+	Orphan      bool
+	Duration    string
+	Regex       string
```

Comment:
Please name this `OrphanIncludeRegex`, as there might be other Regexes in the future, and it works with Orphans anyway.